src/views/createQuestionElement.js

```diff
@@ -7,6 +7,16 @@ const createQuestionElement = (question) => {
     const title = createDOMElement('h1');
     title.innerText = question.text;
     container.appendChild(title);
+   // add links 
+   const link1 = createDOMElement('a');
+   link1.innerText = question.links[0].text;
+   link1.href = question.links[0].href;
+   const link2 = createDOMElement('a');
+   link2.innerText = question.links[1].text;
+   link2.href = question.links[1].href;
+
+   container.appendChild(link1);
+    container.appendChild(link2);
```

Comment:
The code would be nicer and more prepared for change if this was dynamic, using a for loop, instead of hardcoded for 2 links.